spec/support/corpus.rb

```diff
@@ -134,15 +138,33 @@ def install_mutant
         system(%w[bundle])
       end
 
-      # Not in the docs. Number from chatting with their support.
-      CIRCLE_CI_CONTAINER_PROCESSES = 2
+      # The effective ruby file paths
+      #
+      # @return [Array<Pathname>]
+      def effective_ruby_paths
+        paths = Pathname
+          .glob(repo_path.join('**/*.rb'))
+          .sort_by(&:size)
+          .reverse
+
+        paths - excluded_paths
+      end
+
+      # The excluded file paths
+      #
+      # @return [Array<Pathname>]
+      def excluded_paths
+        exclude.flat_map do |pattern|
+          Pathname.glob(repo_path.join(pattern))
+        end
+      end
 
       # Number of parallel processes to use
       #
       # @return [Fixnum]
       def parallel_processes
-        if ENV['CI']
-          Mutant::Config::DEFAULT.jobs
+        if ENV.key?('CI')
```

Comment:
I'd be interested in hearing why mutant shouldn't be CI aware. I would agree for a generic application but mutant is specifically used for testing so I would think that continuous integration would be something it could know about.

Regardless, `MutantSpec.ci?` sounds like a good solution if you don't like `Mutant.ci?`